components/app-configs/mlde-workspace-template/base/kustomization.yaml

```diff
@@ -1,17 +1,15 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-namespace: <replace me>
```

Comment:
Do we actually want to remove the namespace for the entire kustomize context?

Reply:
Nevermind, I should have read the rest of the changes

Reply:
@channolanp You make a good point... moved the namespace to the overlay so the base is more generic

Reply:
Then someone can define the namespace and only apply the base if they want.